packages/bundle/src/adaptiveCards/Attachment/AdaptiveCardAttachment.js

```diff
@@ -41,18 +38,21 @@ export default class AdaptiveCardAttachment extends React.Component {
         card,
         errors
       };
-    });
-  }
+    }
+  }, [adaptiveCards, content]);
 
-  render() {
-    const {
-      props: { adaptiveCardHostConfig, adaptiveCards, attachment, renderMarkdown }
-    } = this;
-    const { card } = this.createAdaptiveCard(adaptiveCards, attachment.content, renderMarkdown);
+  const { card } = createAdaptiveCard;
```

Comment:
This line can be refactored into the `useMemo` function. This is because we never use the `createAdaptiveCard`. In this code, we are only interested in `createAdaptiveCard.card`.

Also, looks like `errors` is not being used. Need some investigations on why. So feel free to open a new bug to track this investigation outside of this PR, because it sounds at least T-shirt size M to me.